docs_src/pipelinesetup.md

```diff
@@ -22,72 +22,33 @@ Model downloader script is automatically called as part of docker-run.sh.  User
 sh modelDownload.sh
 ```
 
+!!! note
+    This may take a while as there are several models to be downloaded for the first time.
+
 !!! note
     To manually download models you can follow links provided in the [Model List](../configs/models/2022/models.list.yml)
 
 ## Step 4: Build the reference design Docker images
 
-You must build the provided component services and create local docker images. To do so, run:
-
-For Intel Core platforms
-```bash
-./docker-build.sh soc
-```
-
-For Intel platforms with Intel discrete GPUs
-```bash
-./docker-build.sh dgpu
-```
+You must build the provided component services and create local docker images. Depending on platforms/hardware you have, refer to the following table to choose one to build:
 
-For Intel platforms with Intel integrated GPUs
-```bash
-cd benchmark-scripts
-make build-benchmark
-make build-igt
-```
+| Platform                                   | Docker Build Command      | Check Success                                |  
+| ------------------------------------------ | ------------------------- |----------------------------------------------|
+| Intel platforms with Intel integrated GPUs | <pre>cd benchmark-scripts<br>make build-benchmark<br>make build-igt</pre> | docker images command to show <b>sco-soc:2.0</b>  |
```

Comment:
build-benchmark just builds the docker images for running the benchmark scripts and tools. We still need to build the dgpu and soc containers. Is that somewhere else?

Reply:
that's a good catch; i mistook the build-benchmark to outputs those; will update it

Reply:
It might make sense to have 2 tables. 

1. for the base image build
2. for the benchmark images using the make file

Reply:
i think we should remove benchmark images here; this is for pipeline setup not the benchmark page; kind of confusion to me to put benchmark in this page

Reply:
good point. We can copy that table into the benchmark.md

Reply:
updated